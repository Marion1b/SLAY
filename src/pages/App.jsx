

function App() {
  return (
    <>
      <h1>SLAY</h1>
    </>
  )
}

export default App
