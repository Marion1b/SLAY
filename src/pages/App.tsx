import LandingPage from "../components/LandingPage";

function App() {
  return (
    <main className="home-page">
      <LandingPage />
    </main>
  )
}

export default App