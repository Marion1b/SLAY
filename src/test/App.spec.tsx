import {render, cleanup} from "@testing-library/react";
import App from "../pages/App";

afterEach(cleanup);

describe("App", ()=> {
    test("should take a snapshot", ()=>{
        const {asFragment} = render(<App />);
        expect(asFragment()).toMatchSnapshot();
    });
});