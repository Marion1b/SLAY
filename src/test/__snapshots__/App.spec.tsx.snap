// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should take a snapshot 1`] = `
<DocumentFragment>
  <main
    class="home-page"
  >
    <section
      class="landing-page"
    >
      <h1>
        SLAY
      </h1>
      <p
        class="subtitle"
      >
        Partagez vos créations musicales, découvrez de nouveaux sons et rencontrez d'autres artistes.
      </p>
      <div>
        <p>
          Découvrir
        </p>
        <svg
          data-file-name="SvgChevronDown"
        />
      </div>
    </section>
  </main>
</DocumentFragment>
`;
